root/client/src/components/table/timeblock.js

```diff
@@ -2,9 +2,13 @@ import { React } from 'react';
 import { calculateTimeFrame } from '../../util/tablehelpers';
 
 export const TimeBlock = ({duration, startTime, classTopPosition, classHeight, handleDelete }) => {
+  const splitStartTime = startTime.split(':');
+  const hour = parseInt(splitStartTime[0]);
+  const minutes = splitStartTime[1];
```

Comment:
why do you have to `parseInt()` the hour but not the minutes?

a possible refactor optimization here is to use destructuring:

```suggestion
  const [hour, minutes] = startTime.split(':');
```

Reply:
I parseInt() the hour because I made a comparison to the number 0, i updated it to deep equal '00' instead.